src/index.ts

```diff
@@ -178,6 +177,7 @@ export type {
   ResumeToken,
   UpdateDescription
 } from './change_stream';
+export { ChangeStream } from './change_stream';
```

Comment:
Why does this need to change? A ChangeStream shouldn't be directly constructed so I think its okay that we only make it accessible via the `watch` calls.